views/about.pug

```diff
@@ -97,7 +97,7 @@ block content
 
             p.
                 In May 2018, we created #[a(href='https://opencollective.com/getbootstrapcdn', rel='noopener', target='_blank') an organization on Open Collective].
-                We use the collective for any money we make from ethical advertisements (#[a(href='https://codefund.io/?utm_campaign=partner&utm_source=bootstrapcdn&utm_medium=bootstrapcdn', rel='noopener', target='_blank') CodeFund])
+                We use the collective for any money we make from ethical advertisements (#[a(href='https://codefund.app/?utm_campaign=partner&utm_source=bootstrapcdn&utm_medium=bootstrapcdn', rel='noopener', target='_blank') CodeFund])
```

Comment:
The only difference is this one, so I'm gonna keep my PR open and close this one.